src/components/base/base-textarea/__tests__/BaseTextarea.spec.ts

```diff
@@ -0,0 +1,135 @@
+import type { ComponentPublicInstance } from 'vue'
+import { mount } from '@vue/test-utils'
+import { describe, it, expect } from 'vitest'
+
+import BaseTextarea from '@/components/base/base-textarea/BaseTextarea.vue'
+import type { ITextareaComponent } from '@/components/base/base-textarea/types'
+
+describe('BaseTextarea', () => {
+  it('should update character count when input changes', async () => {
```

Comment:
este `it` podría estar dentro de `Test user events behaviours` porqué estamos testeando un evento provocado por el usuario